.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
 <!--
 Thanks for contributing a pull request! Please ensure you have taken a look at
 the contribution guidelines: https://github.com/skops-dev/skops/blob/main/CONTRIBUTING.rst
-This guideline contains crucial information for your PR to be merged, such as setting up 
```

Comment:
not sure what happened here, but we've moved this file, you shouldn't be having any changes here.